gobblin-config-management/gobblin-config-core/src/test/java/gobblin/config/store/hdfs/SimpleHdfsConfigureStoreFactoryTest.java

```diff
@@ -44,4 +47,49 @@ public void testGetDefaults() throws URISyntaxException, ConfigStoreCreationExce
       }
     }
   }
+
+
+  @Test
+  public void testConfiguration() throws Exception {
+    FileSystem localFS = FileSystem.getLocal(new Configuration());
+    Path testRoot = localFS.makeQualified(new Path("dir1"));
+    Path configRoot = localFS.makeQualified(new Path(testRoot, "dir2"));
+    Path configStoreRoot = new Path(configRoot,
+                                    SimpleHDFSConfigStore.CONFIG_STORE_NAME);
+    Assert.assertTrue(localFS.mkdirs(configStoreRoot));
```

Comment:
One of the ancestors is :

    finally {
      localFS.delete(testRoot, true);
    }